bugsnag-android-core/src/main/java/com/bugsnag/android/Client.java

```diff
@@ -215,9 +215,6 @@ public void onErrorIOFailure(Exception exc, File errorFile, String context) {
                 metaData.addToTab(INTERNAL_DIAGNOSTICS_TAB, "canWrite", errorFile.canWrite());
                 metaData.addToTab(INTERNAL_DIAGNOSTICS_TAB, "exists", errorFile.exists());
 
-                @SuppressLint("UsableSpace") // storagemanager alternative API requires API 26
```

Comment:
Do we want to continue to store the cache's usable space in addition on our diagnostics tab?

Reply:
You're right, we do. I missed the `getCacheDir()` aspect of this and thought it was equivalent to `DeviceData#calculateFreeDisk()` - I've reinstated this.